packages/@webex/media-helpers/src/index.ts

```diff
@@ -1,19 +1,19 @@
 export {
   getDevices,
-  LocalTrack,
-  LocalDisplayTrack,
-  LocalSystemAudioTrack,
-  LocalTrackEvents,
-  type TrackMuteEvent,
```

Comment:
The new webrtc-core doesn't export an equivalent of `TrackMuteEvent` because `TypedEvent` should already enforce the syntax of the event handler (i.e. TypeScript should fail to compile if the handler doesn't contain the expected arguments).